components/layout/src/index.js

```diff
@@ -17,7 +17,7 @@ const StyledLayout = styled('div')({
   },
 });
 
-const Layout = ({ children }) => <StyledLayout>{children}</StyledLayout>;
+const Layout = ({ children, ...props }) => <StyledLayout {...props}>{children}</StyledLayout>;
```

Comment:
this can be just `const Layout = props => <StyledLayout {...props} />`
as `children` is a prop